src/forms/random_meal_form.php

```diff
@@ -0,0 +1,51 @@
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="card col-sm-6">
+            <div class="card">
+                <div class="card-header">
+                    Hungry?
+                </div>
+                <div class="card-body">
+                    <h5 class="card-title">Don't know what to eat?..</h5>
+                    <p class="card-text">With this sweet meal api at our fingertips, generating meal recipes has never been easier.</p>
+                    <a href="/random_meal.php?showMeal=1" class="btn btn-primary">Show me the light</a>
+                </div>
+            </div>
+            <?php if (isset($meal)) : ?>
+            <div class="card">
+                <h5 class="card-title text-center"><?= $meal['strMeal']; ?></h5>
+                <img class="card-img-top" src="<?= $meal['strMealThumb']; ?>" alt="Card image cap">
+                <div class="card-body">
+                    <h5 class="card-title">Ingredients You Will Need</h5>
+                    <ul>
+                       <?php foreach ($combined as $key => $value) {
+                            echo "<li>$key" . " - " . "$value</li>";
+                       } ?>
+                    </ul>
+                    <h5 class="card-title">Cooking Instructions</h5>
+                    <ul>
+                        <?php $chunks = explode('.', $meal['strInstructions']); ?>
+                        <?php foreach ($chunks as $element) : ?>
+                            <?php if ($element && trim(strlen($element)) > 4) : ?>
+                           <li><?= $element ?></li>
+                        <?php endif; ?>
+                        <?php endforeach; ?>
+                    </ul>
+                </div>
+            </div>
+            <div class="card">
+                <h5 class="card-title">Want to follow a YouTube how to?</h5>
+                <div class="card-body">
+                    <div class="btn-group" role="group">
+                        <a class="btn btn-primary" target="@" href="<?= $meal['strYoutube']; ?>"><?= $meal['strMeal']; ?></a>
+                        <a class="btn btn-primary" href="#top">Back To Top</a>
```

Comment:
These two buttons are side-by-side and the same color. There should be some spacing between these buttons and a more muted color chosen for the `Back to top` since it's not a call-to-action. `default` is a good choice here for that